pkg/testrig/template/dockerfile.go

```diff
@@ -0,0 +1,34 @@
+package template
+
+func GetDockerfileStub() string {
+	return `
+FROM golang:latest as builder
+ENV http_proxy=http://docker.for.mac.host.internal:3128
+ENV https_proxy=http://docker.for.mac.host.internal:3128
+
+WORKDIR /app
+
+# allow insecure download for ANZ root CA certificates, we need them for golang
+RUN git config --global http.sslVerify false
+RUN git clone --depth 1 https://github.service.anz/ocp/ocp-cacerts.git
+RUN cp ocp-cacerts/global/*.crt /usr/local/share/ca-certificates/
```

Comment:
none of the anz internal stuff should be here

Reply:
any suggestions?
without this bit `go mod download` won't be able to work within corp network (we have these installed on workstations on enrolment)

I thought I'm safe to do this as codegen generates stuff with ANZ internal dependencies anyway

Reply:
Let's talk tomorrow in-person. I have a solution for this - the same I am going to propose for the Dockerfile in the sysl repo for sysl itself.

Reply:
In short:
```
# Allow custom GOPROXY in case in network-constrained environment (corporate)
ARG GOPROXY
ENV GOPROXY=${GOPROXY}
```

`docker build --build-arg=GOPROXY=https://artifactory.gcp.anz/artifactory/go -t tag .`

Reply:
great, this way I like it much more